parley_bench/README.md

```diff
@@ -0,0 +1,19 @@
+# Parley Benchmarks
+
+A suite of benchmarks used to evaluate Parley performance.
+
+## Setup
+
+Install `cargo-export` via `cargo install cargo-export`.
```

Comment:
We should use

````suggestion
Install [`cargo-export`](https://crates.io/crates/cargo-export) via: 
```sh
$ cargo install cargo-export
```
````
We could also consider:
````suggestion
Install [`cargo-export`](https://crates.io/crates/cargo-export) via: 
```sh
$ cargo install cargo-export --locked
```
````

But I note that cargo-export itself doesn't suggest to use `--locked`. I'd stand by either suggestion.